### ISSUE TITLE
[1.4 Alpha] Resolution resetting from 1920x1080 to some variation of 1536x864

### ISSUE THREAD

**Issue by Nobisyu12** [bluemagic123/tModLoader#2262]
### Version

Latest 1.4-stable

### OS

Windows

### Platform

Steam

### This bug affects

Gameplay as a Player

### Description

I don't know the exact conditions other than upon startup of the game, but the game likes to reset my resolution from 1920x1080 to 1536x864. It makes everything super blurry, sometimes stretched, and really really big. I am trying to play on Windowed Mode, but even in full screen if I select a different window than tModLoader, it resets the resolution to 1536x864. tModLoader 1.3 or vanilla 1.4 do not have this issue.

### Log File

[client.log](https://github.com/tModLoader/tModLoader/files/8446477/client.log)

### Steps to reproduce

1. Open tModLoader 1.4
2. Resolution has been set to 1536x864

alternate
1. Open tModLoader 1.4
2. Correct resolution in graphics settings to 1920x1080 in fullscreen
3. Focus a different window than tModLoader 1.4
4. Refocus tModLoader 1.4
5. Resolution has reset to 1536x864

alternate
1. Open tModLoader 1.4
2. Correct resolution in graphics settings to 1920x1080 in windowed
3. Game does not fill monitor size but instead sort of cuts itself off and does not display the full display. Sort of like if you zoomed in on the menu (the attached image is the entirety of what you can see on the screen after step 3 is completed)
![image](https://user-images.githubusercontent.com/67725657/162307141-e2c8b015-625c-49d4-a9e3-43dee869f2e0.png)

### Expected Behavior

The game should match the resolution of your monitor and not reset to a strange arbitrary value.

### Actual Behavior

The game resets your resolution to an arbitrary value resulting in in stretched UI or incorrectly sized UI as well as strange effects in windowed mode.

### Reproduction frequency

This issue happens upon loading tModLoader 1.4 every single time

**Comment by direwolf420** [comment]
Does this fix help you? https://github.com/tModLoader/tModLoader/wiki/Basic-tModLoader-Usage-FAQ#screen-is-blurry-or-resolution-resets-to-1536x864
If so, close the issue.

**Comment by Solxanich** [comment]
Per Dire, possible duplicate of #2078 . Pending a confirmation of workaround.

Will close in about two weeks as assumed duplicate unless an otherwise response provided.

**Comment by Solxanich** [comment]
Fixed in #2357 .
Fix is in Preview branch
